### PR TITLE
SQLite3 and CSS problems fixed

### PR DESCRIPTION
**Pull request by dpgraham**
-Added 'electron-rebuild' to fix problem with SQLite3 native module not working with electron
-Some CSS fixes
-Better handle dcaps type changes

Reviewer: @jlipps

### CODE REVIEW

File: app/components/Session/NewSessionForm.js [hunk 1]
```diff
@@ -38,6 +40,44 @@ export default class NewSessionForm extends Component {
...
+  /**
+   * Callback when the type of a dcap is changed
+   */
+  handleSetType (index, type) {
+    let {setCapabilityParam, caps} = this.props;
+    setCapabilityParam(index, 'type', type);
+
+    // Translate the current value to the new type
+    let translatedValue = caps[index].value;
+    typeof('');
```
Comment (jlipps) [inline app/components/Session/NewSessionForm.js]: unclear why we have a bare `typeof` call here?

**Comment by dpgraham** [comment]
I squashed a bunch of the commits together and will rebase the changes. I had to rewrite history on a remote branch and do a forced update but I assume that this is okay when it's your own personal branch.

**Comment by jlipps** [comment]
Yes, rebase + force push on a personal branch is an OK method. Just ensure you're never on master :-) let me know when you're done and I'll have another look.

**Comment by dpgraham** [comment]
Did another push, ready for review again.

**Comment by jlipps** [comment]
looks great, merge away

**Pull request merged**